paradise.dme

```diff
@@ -1264,7 +1262,7 @@
 #include "code\modules\mob\living\simple_animal\friendly\corgi.dm"
 #include "code\modules\mob\living\simple_animal\friendly\crab.dm"
 #include "code\modules\mob\living\simple_animal\friendly\farm_animals.dm"
-#include "code\modules\mob\living\simple_animal\friendly\fox.dm"
+#include "code\modules\mob\living\simple_animal\friendly\Fox.dm"
```

Comment:
Don't think that's supposed to be capitalized. 

Also, the map changes were not mapmerged, causing a merge conflict.